website/developer-docs/index.md

```diff
@@ -121,6 +123,8 @@ This is documented in the [developer docs](./setup/frontend-dev-environment.md)
 
 Contributions to the technical documentation are greatly appreciated. Open a PR if you have improvements to make or new content to add. If you have questions or suggestions about the documentation, open an Issue. No contribution is too small.
 
+Please be sure to refer to our [Style Guide](../developer-docs/docs/style-guide.mdx) for the docs, and use a [template](./docs/templates/index.md) to make it easier for you. The style guidelines are also used for any Integrations documentation, and we have a template for Integrations as well, in our [Github repo](https://github.com/goauthentik/authentik) at `/website/integrations/_template/service.md`.
```

Comment:
You use the phrase `[docs template]` in that link consistently up until this point.  Why change?